com.ibm.streamsx.inet/impl/java/src/com/ibm/streamsx/inet/http/HTTPPostOper.java

```diff
@@ -182,7 +184,17 @@ public void setExtraHeaders(List<String> val) {
 	public void setAcceptAllCertificates(boolean val) {
 		this.acceptAllCertificates = val;
 	}
-	
+	@Parameter(optional=true, 
```

Comment:
Will the SPL compiler validate the attribute names?  That is, I can remove the '...corresponding input attribute are ignored' statement?